docs/spelling_wordlist.txt

```diff
@@ -35,6 +35,7 @@ BaseOperator
 BaseView
 Beauchemin
 Behaviour
+BigQueryHook
```

Comment:
The docs spelling check wasn't recognizing `BigQueryHook` as a valid word, so I added it here